libparsec/crates/platform_device_loader/Cargo.toml

```diff
@@ -45,3 +46,4 @@ libparsec_tests_lite = { workspace = true }
 libparsec_tests_fixtures = { workspace = true, features = [
     "test-with-platform-device-loader-testbed",
 ] }
+hex = { workspace = true }
```

Comment:
This is not needed as `hex!` is already provided as part of `libparsec_tests_fixtures::prelude::*`

Reply:
The `hex` does not provide the macro `hex!` but the function `encode` that is used to generate the blob for the device.
I wanted to keep the dependency listed to simplify updating the blob, but maybe it's not needed

Reply:
I don't thing this is needed since we already have `misc/test_expected_payload_cooker.py` that deals with this in a standardized way.

Reply:
done